doc/misc/npm-config.md

```diff
@@ -278,7 +278,7 @@ This is a list of CIDR address to be used when configuring limited access tokens
 
 ### color
 
-* Default: true
+* Default: true, unless the environment variable `NO_COLOR` is set
```

Comment:
You can remove this. Please mention `NO_COLOR` in the description below, instead, and how to use it. I'd rather keep this describing what the "no action" default is.